GUIDELINES.md

```diff
@@ -0,0 +1,78 @@
+# Bunch-tool common guidelines
+
+Those guidelines cover most common scenarios and generally useful for a usual developer, meaning that they won't help you if
+you have a tough luck of being deeply involved with bunches and support of different IDE versions (in which case, you 
+probably don't need guidance anyways :) )
+
+## What should I do to not break bunch-versions accidentally?
+
+The recommended way is to install pre-commit hook (if you're using CLI for working with Git), or just use IntelliJ IDEA
+support for Git, which has built-in support of bunches.
+
+Another, less convenient way, is to just watch for status of your remote-runs. If your changes introduce some compatibility 
+issues, most likely you'll see failing tests in corresponding configurations (like, `IDEA Plugin tests (183)` for a IDEA 183 
+bunch). 
+
+## I need to make changes in bunch-files, what should I do?
+
+For the sake of example, suppose that:
+
+* you are on a branch `my-feature`, 
+* default bunch is `191`
+* you have made changes to a file `foo.kt`, 
+* this file has a bunch-version `foo.kt.183`, 
+* you know that this bunch versions requires changes too
+
+**Option #1. Plain-text editing**
+You can just edit bunch-versions (`foo.kt.183` in our example) manually, in a plain-text mode, and commit changes to them. 
+This is often enough for simple changes and/or small bunch-files. 
+
+
+> Tip: you can use `Compare With...` (Ctrl+D/Cmd+D by default) in IntelliJ IDEA to see visual diff of files
+
+
+**Option #2. Full-switch + cherry-pick changes automatically**
+For more complicated changes, you're usually would like to have a proper highlighting and other IDEA features in bunch 
+files. In this case, you can try following workflow:
+
+* create and checkout new branch, e.g. `git checkout -b my-feature-183`
+* Switch to the needed bunch: `bunch switch . 183
```

Comment:
This and following preformatted blocks stick together, so it looks like `bunch switch . 183 foo.kt` in result.